package.json

```diff
@@ -5,7 +5,7 @@
     "email": "azsdkteam@microsoft.com",
     "url": "https://github.com/Azure/ms-rest-js"
   },
-  "version": "0.18.0",
+  "version": "0.19.0",
```

Comment:
Ahh, forgot to update the changelog again. I might just grind through the old versions to fill out the changelog so we can start making a habit of it.